csrankings-j.csv

```diff
@@ -1083,6 +1083,7 @@ Jie Fu,Worcester Polytechnic Institute,https://www.wpi.edu/academics/facultydir/
 Jie Gao 0001,Rutgers University,https://sites.rutgers.edu/jie-gao,P1CMmgEAAAAJ
 Jie Gu 0001,Northwestern University,http://users.eecs.northwestern.edu/~jgu,HP-yzF0AAAAJ
 Jie Hao,BUPT,https://scss.bupt.edu.cn/info/1063/3808.htm,z26c40QAAAAJ
+Jie Hou,Saint Louis University,https://cs.slu.edu/~hou/,RsC9b_wAAAAJ

```

Comment:
This is just a disambiguation page (https://dblp.org/pid/17/36.html). Please remove and follow the instructions to get this faculty member added to DBLP.